src/doc/using_visit/Quantitative/XRayImageQuery.rst

```diff
@@ -788,11 +792,20 @@ Why Conduit Output?
 """""""""""""""""""
 
 Conduit `Blueprint <https://llnl-conduit.readthedocs.io/en/latest/blueprint.html>`_ output types were added to the X Ray Image Query primarily to facilitate usability and convenience.
-Before Conduit Blueprint formats were available as output types, the X Ray Image Query would often produce large numbers of output files, particularly when using the bov or rawfloats output type, which was a popular choice because it provided the raw data.
+Before Conduit Blueprint formats were available as output types, the X Ray Image Query would often produce large numbers of output files, particularly when using the BOV or rawfloats output type, which was a popular choice because it provided the raw data.
```

Comment:
Just one quick observation. I think the information in these paragraphs is important motivation for the current desgin to have somewhere. I just don't think its something any new (or, maybe even regular) user of VisIt is going to need to know.

IMHO, what needs to exist in the `using_visit` part of the manual is what users need to know to use the current version of VisIt.

Our documentation is already a lot to get through and so I think we should take care not to include information, though important in some way, they don't need to be exposed to in order to use the current interface.

So, my suggestion is to remove or wholly reduce to a single sentence lines 795...803.

Maybe just say something like...

*The output of the XRay Image query is designed to be all in one file and behave like any other high-level VisIt_ database containing a variety of useful outputs from the operation.*

Reply:
Maybe the right place for the kind of information in lines 795...803 is release notes?

Reply:
Hmmm. I wanted to put this into the "Why Conduit" section to help motivate why users ought to use the conduit output choices as opposed to the other legacy choices. We still support them because there is no reason not to, but the newer conduit choices have a lot of advantages. I think it is important for this information to be available somewhere in the documentation for the query so users can be aware if they are using the legacy options why they should switch over.

Reply:
Ah...GitHub's viewer truncated the display just before the section heading and so I never saw that. That makes more sense then so I feel like my remarks are not so applicable.

That said, if this method is superior to others, why do we need to continue supporting the others? I think there is a reason NOT to support the other formats...reduced variations we need to support, maybe reduced code we need to support too.

Reply:
I wouldn't be opposed to removing the older output types but I would want to run that by @cyrush first and make sure that wouldn't mess up user workflows that rely on them. One issue is that we currently (as far as I know) allow conduit to be optionally installed, so if a user is using an install without conduit and there are no non-conduit output types, they are hosed if they try to use the x ray image query and we have removed all the non-conduit options.

Food for thought that is beyond the purview of this particular PR. I'll make a note in my x ray query ticket.